File: web3rpc/rpc-specs/code-samples/javascript/debug/logging/verbosity.js
const { JsonRpcProvider } = require("@klaytn/ethers-ext");
(() => {
let sdk = new JsonRpcProvider("https://public-en-baobab.klaytn.net");    
    const level = 3;

    sdk.debug.verbosity(level, {}, (err, data, response) => {
        console.log(data);
    });
}
)()

File: web3rpc/rpc-specs/code-samples/python/debug/logging/verbosity.py
from web3 import Web3
from web3py_ext import extend

host = "https://api.baobab.klaytn.net:8651"

level = 3

w3 = Web3(Web3.HTTPProvider(host))
debug_response = w3.debug.verbosity(level)

print(debug_response)
